sentinel-dashboard/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM amd64/buildpack-deps:buster-curl as installer
+
+ARG SENTINEL_VERSION
+
+RUN set -x \
+    && curl -SL --output /home/sentinel-dashboard.jar https://github.com/alibaba/Sentinel/releases/download/${SENTINEL_VERSION:-1.8.6}/sentinel-dashboard-${SENTINEL_VERSION:-1.8.6}.jar
```

Comment:
maybe something wrong with the grammar

<img width="1767" alt="image" src="https://user-images.githubusercontent.com/43714056/203906292-1a53c8b0-8037-4811-bb5f-16d590e22510.png">

Reply:
<img width="1520" alt="image" src="https://user-images.githubusercontent.com/1269496/203906971-63daa03f-1b9e-4fa3-8c00-01ae6d154cdf.png">

It works fine in my local env, do you have suggestion to change it?

Reply:
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/1269496/203907934-2f56b548-2477-46f1-94f2-f72bc9161ea5.png">

With build arg also works.

Reply:
you can set the default value in the declare code like following:
``` shell 
ARG SENTINEL_VERSION=1.8.6
```

Reply:
<img width="1723" alt="image" src="https://user-images.githubusercontent.com/43714056/203911881-3031d4b7-5061-4f09-a547-0652517bf096.png">

Reply:
Because the 2nd screen is in build progress, after build log like below:
<img width="1428" alt="image" src="https://user-images.githubusercontent.com/1269496/203917842-f49dfa02-6ce7-4b49-83b9-07a9eb2d7d22.png">


You can try it in your local, the docker build log designed like this.

Reply:
I  has tried it again, and it works success this time. :smiley:

But I still think the default value should be given in the declare. when sentinel bump into next version, we only need to change it in the declare, so that can prevent omission changes.

Reply:
Yes, that is also a good way, let me update it.

Reply:
Done.